### ISSUE TITLE
Illuminate\Database\QueryException: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'mdoughty@lincoln.ac.uk' fo...

### ISSUE THREAD

**Issue by marc-hanheide** [LCAS/pgr_system#62]
https://sentry.io/university-of-lincoln/pgr-system/issues/701584087/

```
PDOException: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'mdoughty@lincoln.ac.uk' for key 'users_university_email_unique'
  File "app/Http/Controllers/StaffController.php", line 128, in store
    'user_type' => 'Staff',
  File "app/Http/Middleware/SamlAuth.php", line 26, in handle
    return $next($request);
  File "app/Http/Middleware/DevUser.php", line 25, in handle
    return $next($request);
  File "app/Http/Middleware/RoutePrefixProxy.php", line 25, in handle
    return $next($request);
  File "public/index.php", line 52
    $request = Illuminate\Http\Request::capture()
...
(73 additional frame(s) were not displayed)

Illuminate\Database\QueryException: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'mdoughty@lincoln.ac.uk' for key 'users_university_email_unique' (SQL: insert into `users` (`first_name`, `last_name`, `university_id`, `university_email`, `user_type`, `updated_at`, `created_at`) values (Mark, Doughty, mdoughty, mdoughty@lincoln.ac.uk, Staff, 2018-09-26 09:51:11, 2018-09-26 09:51:11))
  File "app/Http/Controllers/StaffController.php", line 128, in store
    'user_type' => 'Staff',
  File "app/Http/Middleware/SamlAuth.php", line 26, in handle
    return $next($request);
  File "app/Http/Middleware/DevUser.php", line 25, in handle
    return $next($request);
  File "app/Http/Middleware/RoutePrefixProxy.php", line 25, in handle
    return $next($request);
  File "public/index.php", line 52
    $request = Illuminate\Http\Request::capture()
...
(71 additional frame(s) were not displayed)
```

**Comment by marc-hanheide** [comment]
@pet1330 I'm still confused as to how this can happen...

**Comment by pet1330** [comment]
This can occur when an Admins miss spell a users name, as it tries to match all of the attributes when finding that user, rather than just their university email. This has now been fixed in: 1412230
